doc/nrf/releases/release-notes-latest.rst

```diff
@@ -80,10 +94,45 @@ The following list summarizes the most important changes inherited from upstream
     * Reimplemented the logger glue layer for better performance.
     * Updated the OpenThread thread priority class to be configurable.
     * Added several Kconfig options to customize the OpenThread stack.
+    * Updated the OpenThread revision to upstream commit ``ac86fe52e62e60a66aeeb1c905cb1294709147e9``
+    * radio: Added sleep to tx as hw radio capability.
+    * net: Added config for OpenThread version selection (either 1.1 or 1.2).
+    * radio: Removed retransmissions from radio capability.
+    * Added config options for NCP vendor hooks, see :option:`CONMFIG_OPENTHREAD_NCP_VENDOR_HOOK_SOURCE`.
+    * Allow use of custom mbedtls
+    * platform: Removed double-buffering in UART send.
+    * Fix the network initialization when :option:`CONFIG_NET_CONFIG_MY_IPV6_ADDR` is not set
+    * Allow to build Zephyrs OpenThread port from user's custom sources, see :option:`CONFIG_OPENTHREAD_SOURCES`.
+    * Allow to compile Diagnostic functions support, see :option:`CONFIG_OPENTHREAD_DIA`.
 
   * Socket offloading:
 
     * Removed dependency to the :option:`CONFIG_NET_SOCKETS_POSIX_NAMES` configuration option.
+    * Fixed Network interface was missing for offloaded drivers (#27037).
+
+  * PPP:
```

Comment:
I don't think we need to mention PPP changes here, it's not something we use with Nordic HW.